modules/javafx.graphics/src/main/java/javafx/css/Match.java

```diff
@@ -77,7 +76,7 @@ public Selector getSelector() {
      * Gets the pseudo class state.
      * @return the pseudo class state
      */
-    public PseudoClassState getPseudoClasses() {
+    public Set<PseudoClass> getPseudoClasses() {
```

Comment:
does this change need a CSR?

Reply:
Yes, thanks for checking. I already flagged it as needing a CSR.